installer/conf/container-health.conf

```diff
@@ -68,7 +68,7 @@
 </match>
 
 
-<match oms.api.KubeHealth.DaemonSet**>
+<match kubehealth.DaemonSet**>
```

Comment:
when you go to IP discovery route (ignite changes i guess) change the below host in out_forward to IP as well)